src/pages/accruals/Accruals.jsx

```diff
@@ -57,24 +57,67 @@ const Accruals = () => {
   const setAccrualsData = (fetchedAccruals) => {
     let accrualsList = [];
     fetchedAccruals.forEach((accrual) => {
-      if (accrual.accrualTypeId === accrualsTypeIds.annualTargetHours) {
-        const targetHours = createAccrualObject(
-          accrual,
-          'annualTargetHours.remainingHoursTitle'
-        );
-        accrualsList.push(targetHours);
-      } else if (accrual.accrualTypeId === accrualsTypeIds.nightHours) {
-        const nightHours = createAccrualObject(
-          accrual,
-          'nightHours.remainingHoursTitle'
-        );
-        accrualsList.push(nightHours);
+      let accrualTitle = '';
+      let accrualOrder = 0;
+
+      switch (accrual.accrualTypeId) {
```

Comment:
Instead of the switch, the title could be stored in accrualTypes constant as well; for example:
`  annualTargetHours: {
    id: 'e502eebb-4663-4e5b-9445-9a20441c18d9',
    title: 'annualTargetHours.remainingHoursTitle',
    displayOrder: 1,
  },
`
Then both the title and the display order could be retrieved as follows:
`      let accrualTitle =
        Object.values(accrualsTypes).find(
          (element) => element.id === accrual.accrualTypeId
        )?.title ?? 'accrualsData.unknownAccrual';
      let accrualOrder =
        Object.values(accrualsTypes).find(
          (element) => element.id === accrual.accrualTypeId
        )?.displayOrder ?? 11;
`
This would make the code less verbose